browser-based-ingestion/README.md

```diff
@@ -0,0 +1,249 @@
+### KVS Browser based Ingestion is used to stream video from the system's webcam to the KVS backend directly without any proxies. ###
+
+Currently, it is only available via Chrome. 
+There are four important tasks that this repository helps perform
+1. Obtaining the video from the webcam/ uploading video file
+2. Converting it to containers and codecs expected by the playback
+3. Sending to the putMedia API via POST calls
+4. Viewing the stream on the browser
+
+
+### Setup ###
+To set-up the project and run the example index.html and index.js files, follow the steps below:
+1. Create a webpack project referring to https://webpack.js.org/guides/getting-started/
+    ```
+    mkdir webpack-demo
+    cd webpack-demo
+    npm init -y
+    npm install webpack webpack-cli --save-dev
+    ```
+
+2. Create dist and src folders in webpack-demo
+    ```
+    mkdir dist
+    mkdir src
+    ```
+
+3. Add webpack.config.js with the following content to the src folder
```

Comment:
Do you need these steps for `webpack.config.js` if you are anyways adding the file to the repo?

Reply:
Removed this part since I changed the way the project would be setup